edx_rest_framework_extensions/auth/jwt/tests/test_decoder.py

```diff
@@ -177,6 +178,32 @@ def test_required_claims(self, claim):
             # Decode to see if MissingRequiredClaimError exception is raised or not
             jwt_decode_handler(token)
 
+    def test_failure_decode_symmetric_set_as_False(self):
+        """
+        Verifies the function logs decode failures with symmetric token set as false,
+        and raises an InvalidTokenError if token is symmetric
+        """
+        # Create valid token symmetric jwt token and set decode_symmetric_token as False.
+        with mock.patch('edx_rest_framework_extensions.auth.jwt.decoder.logger') as patched_log:
+            with self.assertRaises(jwt.InvalidTokenError):
+                token = generate_jwt_token(self.payload)
+                jwt_decode_handler(token, decode_symmetric_token=False)
+
+            patched_log.exception.assert_any_call("Token verification failed.")
```

Comment:
I don't think we need this. I'd say we just test this via the other new test. This is just my opinion though. Feel free to disagree.